src/extensions/unitstable/Units/TableWidget.tsx

```diff
@@ -0,0 +1,195 @@
+import { faCaretDown, faCaretUp } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Split UnitsTable into two components. One is TableWidget, a more generic widget that can sort columns.